src/sidebar/services/groups.js

```diff
@@ -51,7 +51,7 @@ function groups($rootScope, store, api, isSidebar, localStorage, serviceUrl, ses
     // If service groups are specified only return those.
     // If a service group doesn't exist in the list of groups don't return it.
     if (svc && svc.groups) {
-      const focusedGroups = groups.filter(g => svc.groups.includes(g.id));
+      const focusedGroups = groups.filter(g => svc.groups.includes(g.id) || svc.groups.includes(g.groupid));
```

Comment:
This line is getting a bit long. Perhaps wrap like so:

```suggestion
      const focusedGroups = groups.filter(g =>
        svc.groups.includes(g.id) || svc.groups.includes(g.groupid)
      );
```